[HaveIBeenPwnedBridge] Add item limit parameter, set default limit to 20

VerifiedJoseph opened the pull request:
This pull request adds the parameter `item_limit` that allows the user to control the number of items returned by bridge. Default limit is 20.

VerifiedJoseph committed:
Limit number of feed items to 20

triatic commented:
20 is an acceptable default limit, but I think things like this should be user-definable.

somini commented:
To implement @triatic's proposal, add a new parameter:

```php
'items_limit' => array(
   'name' => 'Limit the number of returned items',
   'type' => 'number',
   'defaultValue' => 20,
)
```

And use it, instead of the hardcoded `20`:

```php
$this->getInput('items_limit')
```

VerifiedJoseph committed:
Add 'item_limit' parameter

Add `item_limit` parameter to allow user to control number of item returned by bridge. Suggested by @triatic and @somini (code).

VerifiedJoseph committed:
Format file with phpcbf

teromene commented:
Merged, thanks !
